foundry/main.rs

```diff
@@ -60,16 +61,47 @@ fn run() -> Result<(), String> {
         .about("Foundry client")
         .subcommand(SubCommand::with_name("commit-hash").about("Print the commit hash of the source tree"));
 
+    let app = app.arg(
+        clap::Arg::with_name("module_arguments")
+            .short("D")
+            .help("Override config values in app-descriptor. For example  '-D foo=bar -D x=y'")
+            .takes_value(true)
+            .multiple(true),
+    );
+
     let matches = Config::parse_args(app);
     let config_path = matches.value_of("config").unwrap_or("./config.ini");
     let conf = Config::parse_file(&matches, config_path).expect("read config file");
 
+    let module_arguments = parse_module_args(matches.values_of("module_arguments"));
+
     match matches.subcommand_name() {
         Some(_) => run_subcommand(&matches),
-        None => run_node(conf, None),
+        None => run_node(conf, module_arguments, None),
     }
 }
 
+fn parse_module_args(args: Option<Values>) -> BTreeMap<String, String> {
+    let mut ret = BTreeMap::new();
+
+    if args.is_none() {
+        return ret
+    }
+
+    let vals: Vec<&str> = args.unwrap().collect();
+    for keyvalue in vals {
+        let splitted: Vec<&str> = keyvalue.split('=').collect();
+        if splitted.len() != 2 {
```

Comment:
It does no harm to permit `=` in a value. One might need to specify `=`s in the value.